tutorials/best_practices/godot_notifications.rst

```diff
@@ -84,6 +84,24 @@ implementing a Timer-timeout loop is another option.
             print("This block runs every 0.5 seconds")
         )
 
+ .. code-tab:: csharp
+
+    using Godot;
+
+    public partial class MyNode : Node
+    {
+        // Allows for recurring operations that don't trigger script logic
+        // every frame (or even every fixed frame).
+        public void _Ready()
```

Comment:
The `_Ready` method is virtual, in C# you need to use the `override` keyword when overriding methods.

```suggestion
        public override void _Ready()
```